label_studio/core/label_config.py

```diff
@@ -63,7 +65,10 @@ def _get_parent_output_tag_name(tag, outputs):
             if name in outputs:
                 return name
 
-    xml_tree = etree.fromstring(config_string)
+    try:
+        xml_tree = etree.fromstring(config_string)
+    except etree.XMLSyntaxError as e:
+        raise LabelStudioXMLSyntaxErrorSentryIgnored(str(e))
```

Comment:
don't we lose a traceback here?

Reply:
working as usual
![image](https://user-images.githubusercontent.com/501780/129776755-cd7c7a08-9155-421f-9386-32738e155475.png)
![image](https://user-images.githubusercontent.com/501780/129776796-ab482127-5e20-4822-9aaa-7f6c08e617e8.png)